### ISSUE TITLE
EncounterNotFound & EncounterPokemonFled

### ISSUE THREAD

**Issue by QustomiZed** [Ar1i/PokemonGo-Bot#849]
I get both of this error.

**Comment by Bikkstar123HD** [comment]
sameeee

**Comment by joeisme90** [comment]
we all do

**Comment by eLDeMe** [comment]
I get it too in 2 accounts, for 10 min, then all went to normal

**Comment by sargathus** [comment]
same here

**Comment by Ar1i** [comment]
If its not fixed, tell me.

**Comment by FikryAZ** [comment]
error reloading pokemon list, older version not error like this

**Comment by Ar1i** [comment]
Need more information, post a screenshot orso please

**Comment by mwaheed92** [comment]
Hello,

Exception is thrown, there is a missing "Configs" directory in Latest Build - 02.08 11.50
Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path '*\PokemonGo-Bot-master\Builds Only\Latest Build - 02.08 11.50\Configs\Config.txt'.

Kindly check it.

**Comment by rihanboo** [comment]
so the update did not fix, and ive tried it all from restarting to reinstalling, teleporting to another location, waiting to be soft then unbanned, nothing worked. I've just been using an old build that works for me, but i really wanted to use the new features that regularly get added in.

I finally got it mine to work and I no longer recieve  the encounternotfound or encounternotinrange like in was getting. what i did was downloaded a program called nox which is an android emulater that allows someone to "wasd" around in pokemon go. I moved a couple blocks away and reset the location to the new position in PokemonGo-Bot.

Voila now its working flawlessly. hope this helps anyone with the encounternotfound/notinrange errors.
